src/auto/tls_connection.rs

```diff
@@ -0,0 +1,377 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ ea993ed)
+// from gir-files (https://github.com/gtk-rs/gir-files @ 469db10)
+// DO NOT EDIT
+
+use Cancellable;
+use Error;
+use IOStream;
+use TlsCertificate;
+use TlsCertificateFlags;
+use TlsDatabase;
+use TlsInteraction;
+use TlsRehandshakeMode;
+use ffi;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct TlsConnection(Object<ffi::GTlsConnection, ffi::GTlsConnectionClass>): IOStream;
```

Comment:
While we're at it, this is useless without GTlsClientConnection and/or GTlsServerConnection. Both of which look trivially autogenerateable :)

Reply:
Done